RoddyCore/src/de/dkfz/roddy/execution/jobs/Job.groovy

```diff
@@ -529,12 +535,12 @@ class Job extends BEJob<BEJob, BEJobResult> {
 
         //Execute the job or create a dummy command.
         if (runJob) {
-            runResult = jobManager.runJob(this)
+            runResult = jobManager.submitJob(this)
             appendToJobStateLogfile(jobManager, executionContext, runResult, null)
             cmd = runResult.command
-            jobDetailsLine << " => " + cmd.getExecutionID()
+            jobDetailsLine << " => " + cmd.setJobID()
```

Comment:
This name looks like a setter! Please rename to something like getSetJobID or something more readable.